### DISCUSSION TITLE
Moving to Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by kuznetsss** [XRPLF/clio#1350] [Polls]
Hi,

We would like to update our minimal supported Ubuntu version to 22.04. The reason for this is that we have migrated to gcc 12.3 and currently we have to compile it from source to install it. 

We understand that some of users might be running Clio on Ubuntu 20.04 and for them such upgrade would be very inconvenient for them. So we would like you to share the Ubuntu version you are running Clio on. If there will be a significant amount of users who wouldn't want the Ubuntu to be upgraded, we will try to stay on 20.04 as long as possible. 

Please vote in the poll below so we could make a decision based on community's opinion.

Thanks!

**Comment by kennyzlei** [comment]
I imagine there are benefits in the migration to gcc 12.3, can you possibly share the benefits of that gcc version upgrade for Clio?

**Reply by kuznetsss** [reply]
Migration to gcc 12.3 allows us to move to C++ 23 and use the latest C++ features like `std::expected`. It doesn't directly affect users or node operators but it improves Clio development process.
